tests/bulk_create/tests.py

```diff
@@ -199,6 +201,67 @@ def test_bulk_insert_expressions(self):
         bbb = Restaurant.objects.filter(name="betty's beetroot bar")
         self.assertEqual(bbb.count(), 1)
 
+    @skipUnlessDBFeature('has_bulk_insert')
+    def test_bulk_insert_nullable_fields(self):
+        # check that NULL can be mixed with other values in nullable fields
```

Comment:
No need for "check that" prefix. Stating the expected behavior is enough since all tests check/test things.